src/main/kotlin/no/nav/tiltaksarrangor/repositories/model/DeltakerlisteDbo.kt

```diff
@@ -9,9 +9,21 @@ data class DeltakerlisteDbo(
 	val navn: String,
 	val status: DeltakerlisteStatus,
 	val arrangorId: UUID,
-	val tiltakNavn: String,
+	private val _tiltakNavn: String,
 	val tiltakType: String,
 	val startDato: LocalDate?,
 	val sluttDato: LocalDate?,
 	val erKurs: Boolean
-)
+) {
+
+	val tiltakNavn = _tiltakNavn
+		get(): String {
+			return when (tiltakType) {
```

Comment:
Jeg synes det er fint at de rensketed navnene ikke lagres, men jeg tror vi bør ha samme logikk i de forskjellige appene.

Måten vi ble enige om å gjøre det tidligere var å sjekke på navnet isteden for type, selv om det ikke føles like naturlig som å sjekke på typen så vil vi ikke overskrive eventuelle endringer på navn fra valp.

Slik er de konvertert i aktivitetskort appen:
```kotlin
fun cleanTiltaksnavn(navn: String) = when (navn) {
	"Arbeidsforberedende trening (AFT)" -> "Arbeidsforberedende trening"
	"Arbeidsrettet rehabilitering (dag)" -> "Arbeidsrettet rehabilitering"
	"Digitalt oppfølgingstiltak for arbeidsledige (jobbklubb)" -> "Digitalt oppfølgingstiltak"
	"Gruppe AMO" -> "Arbeidsmarkedsopplæring"
	else -> navn
}
```